gradle/libs.versions.toml

```diff
@@ -81,6 +81,7 @@ opentelemetry-bom = { module = "io.opentelemetry:opentelemetry-bom", version = "
 opentelemetry-semconv = { module = "io.opentelemetry.semconv:opentelemetry-semconv", version = "1.25.0-alpha" }
 picocli = { module = "info.picocli:picocli-codegen", version.ref = "picocli" }
 picocli-codegen = { module = "info.picocli:picocli-codegen", version.ref = "picocli" }
+postgresql = { module = "org.postgresql:postgresql", version = "42.7.4" }
```

Comment:
Not needed

Reply:
Again, this is needed. Otherwise the resulting Docker images will be unusable.

Reply:
I mean, we can remove this and ship just with H2 for now. But I don't see the point of a Polaris Docker image that is only usable with H2.

Reply:
This declaration does look now (after recent review-driven changes)